packages/start/entry-client/StartClient.tsx

```diff
@@ -1,21 +1,16 @@
 import { MetaProvider } from "solid-meta";
 import { Router } from "solid-app-router";
-import { StartProvider } from "./StartContext";
+import { StartProvider } from "../entry-server/StartContext";
 // @ts-expect-error
 import Root from "~/root";
-import { createSignal } from "solid-js";
 
 const rootData = Object.values(import.meta.globEager("/src/root.data.(js|ts)"))[0];
 const dataFn = rootData ? rootData.default : undefined;
-const [request, setRequest] = createSignal(new Request("http://localhost:3000/"));
 
 export default () => {
   return (
     <StartProvider
-      context={{
-        responseHeaders: new Headers(),
-        request: request()
-      }}
+      context={{}}
```

Comment:
This has typescript issues, maybe mark as any or @ts-expect-error, but I think we should stub the values out with reasonable defaults. Though probably even better to make it a proxy and throw on the Client if the user tries to access it on the client.